drake/util/test/testDrakeGradientUtil.cpp

```diff
@@ -3,7 +3,6 @@
 #include "drake/util/testUtil.h"
 #include <Eigen/Core>
 #include <Eigen/Geometry>
-#include <unsupported/Eigen/KroneckerProduct> // unsupported...
```

Comment:
There was an issue with KroneckerProduct; decided to just remove the test that needed it, since it's testing matGradMultMat, which has basically been replaced by multiplying AutoDiff matrices.